player/public/coreplugins/org.ekstep.epubrenderer-1.0/renderer/plugin.js

```diff
@@ -13,10 +13,13 @@ org.ekstep.contentrenderer.baseLauncher.extend({
     lastPage: false,
     stageId:[],
     enableHeartBeatEvent: false,
+    mimeType: ['application/epub'],
     initLauncher: function () {
-        EkstepRendererAPI.addEventListener('content:load:application/vnd.ekstep.epub-archive', this.launch, this);
+        EkstepRendererAPI.dispatchEvent('renderer:launcher:register', this);
```

Comment:
Clean & register should be in baseLaunher plugn itself. If any plugins wants to override clean, they can override by calling this.super().